helion/_compiler/indexing_strategy.py

```diff
@@ -394,11 +702,88 @@ def codegen_store(
                 store_val=store_value,
             )
 
+        if _use_epilogue_subtile():
+            subtile_value = self.codegen_store_subtile(
+                state,
+                fake_tensor,
+                indexing,
+                store_value,
+                extra_mask=None,
+                subscript=None,
+            )
+
+            if subtile_value:
+                return subtile_value
+            (store_value,) = _apply_pointwise_to_subtile(state, [store_value])
+
         return expr_from_string(
             f"{indexing.tensor_descriptor(state)}.store({indexing.offsets_str_permuted(state)}, {{value}})",
             value=store_value,
         )
 
+    def codegen_store_subtile(
```

Comment:
This looks like a lot of duplicated code.  We should refactor things to share more.